playbooks/roles/local.defaults/templates/config.yml.j2

```diff
@@ -70,5 +70,5 @@ config:
     {%- endfor +%}
   {%- endfor +%}
 
-  statedir: "{{ misc.host.statedir }}/sit.{{ be }}_statedump"
+  statedir: "{{ misc.host.statedir }}/sit.{{ backend }}_statedump"
```

Comment:
Why not just use a constant name ? is it necessary to include the backend name ?

If we develop tools that automatically scan the data, it will need to know the backend before hand to easily access it. With a constant name, the tool can directly find the information and deduce the backed, if necessary, from the contents of the directory.

Reply:
> Why not just use a constant name ? is it necessary to include the backend name ?

I think its better to have it named after the backend used to easily differentiate in a first pass by just looking at the top-level directory name.

> If we develop tools that automatically scan the data, it will need to know the backend before hand to easily access it. With a constant name, the tool can directly find the information and deduce the backed, if necessary, from the contents of the directory.

Let's keep the backend(and/or variant) name until we have something along the lines mentioned above.